docs/topics/OAuth2.md

```diff
@@ -253,7 +253,7 @@ Bot authorization is a special server-less and callback-less OAuth2 flow that ma
 ###### URL Example
 
 ```
-https://discord.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=1
+https://discord.com/api/oauth2/authorize?client_id=157730590492196864&scope=bot&permissions=0
```

Comment:
Perms: 1 doesn't even exist lol.

Reply:
I am pretty sure that permission `1` / `CREATE_INSTANT_INVITE` exists; see https://discord.com/developers/docs/topics/permissions.

It also gets listed when adding the bot: 

![image](https://user-images.githubusercontent.com/46497296/98127225-bd13ba80-1eb6-11eb-86c9-4231d028fa63.png)